src/components/Card.js

```diff
@@ -3,18 +3,23 @@ import React from "react"
 import b from "../images/back.svg"
 
 function Card(props) {
-  const {card, front = true, back, Component, height} = props
-  const style = {height}
+  const {card, front = true, back, Component, height, style, className} = props
```

Comment:
Here, it's better to assign default values to make the logic below simpler.

1. `const {card, front = true, back, Component, height, style = {}, className = ""} = props`
2. `style.height = height`
3. `if ... else` can be removed